app/views/article_recruit_mailer/article_recruit_email.html.haml

```diff
@@ -0,0 +1,7 @@
+%h2="Dear Author: Oregon State University Libraries invites you to deposit manuscript of recent publication for open access"
+%hr
+- @publication.web_of_science_source_record.record.titles.each do |t|
+  %h3= t
+
+Attach manuscript with this link:
+= link_to @publication.web_of_science_source_record[:uid], publication_url(@publication)
```

Comment:
this should be `claim_url(@publication.web_of_science_source_record[:hashed_uid])`